src/emulation/promises.ts

```diff
@@ -9,7 +9,7 @@ import type { ReadableStreamController } from 'stream/web';
 import { Errno, ErrnoError } from '../error.js';
 import type { File } from '../file.js';
 import { isAppendable, isExclusive, isReadable, isTruncating, isWriteable, parseFlag } from '../file.js';
-import type { FileContents } from '../filesystem.js';
+import type { FileContents, FileSystem } from '../filesystem.js';
```

Comment:
Correct me if I'm wrong, but I don't see any changes that would require importing `FileSystem`.